vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp

```diff
@@ -29,8 +30,9 @@ $( document ).ready(function() {
 	
 	$("#addSpaceLinkButton").click(function(e) {
 		$("#bgImage").on("click", function(e){
-		    e.preventDefault();
-		    var icon = $('<span data-feather="navigation-2" class="flex"></span>');
+			e.preventDefault();			
```

Comment:
indentation?

Reply:
Corrected the indentation in my second Task.

Reply:
I Fixed this as part of the second task